src/libcore/convert.rs

```diff
@@ -50,20 +50,41 @@
 
 use str::FromStr;
 
-/// A cheap, reference-to-reference conversion.
+/// A cheap reference-to-reference conversion. Used to convert a value to a
+/// reference value within generic code.
```

Comment:
I wonder if this isn't too long for a summary line; it might get cut off on https://doc.rust-lang.org/stable/std/convert/index.html. Have you checked? if not, maybe

> A cheap reference-to-reference conversion, used within generic code.